sql/core/src/test/resources/sql-tests/results/operators.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 38
+-- Number of queries: 48
```

Comment:
This fix is not related to this pr though, this is not updated in https://github.com/apache/spark/commit/3c9eef35a85fd841efb4491b299345a2edf30e87